src/CMakeLists.txt

```diff
@@ -15,5 +15,20 @@
 # specific language governing permissions and limitations
 # under the License.
 
+add_subdirectory(arrow)
 add_subdirectory(core)
 add_subdirectory(puffin)
+
+include(CMakePackageConfigHelpers)
+
+configure_package_config_file("${CMAKE_CURRENT_SOURCE_DIR}/config.cmake.in"
+                              "${CMAKE_CURRENT_BINARY_DIR}/iceberg-config.cmake"
```

Comment:
If we use `Iceberg` as a CMake package name, `IcebergConfig.cmake` will be better for consistency.
(`iceberg-config.cmake` works too.)

Reply:
I saw that both `<lowercasePackageName>-config.cmake` and `<PackageName>Config.cmake` are acceptable. It seems that the official CMake documentation uses a lot of the former style as examples and some package managers (e.g. Conan) also use the former one. Therefore I follow the same pattern.

Reply:
Finally, I chose to follow what Apache Arrow does...